src/pages/Facility/services/serviceRequests/components/AddMultipleChargeItemsSheet.tsx

```
@@ -208,6 +227,18 @@ export default function AddMultipleChargeItemsSheet({
                       </Button>
                     </div>
 
+                    <div className="space-y-1">
+                      <label className="text-sm text-gray-500">
+                        {t("performer")}
+                      </label>
```

Biome (2.1.2)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)